sdk/core/azure-core/src/main/java/com/azure/android/core/http/interceptor/LogUtils.java

```diff
@@ -16,7 +19,7 @@
      * Attempts to retrieve and parse the Content-Length header into a numeric representation.
      *
      * @param headers HTTP headers that are checked for containing Content-Length.
-     * @return long value indicating the content length of the Request or Response
+     * @return Long value indicating the content length of the Request or Response
```

Comment:
```suggestion
     * @return The content length of the Request or Response
```
The type is encoded in the signature, let's not bother duplicating that in the documentation.